src/toolbar/ToolBar.js

```diff
@@ -119,7 +119,9 @@ class ToolBar extends React.Component{
             <i className="glyphicon glyphicon-trash"></i> Delete
           </button>:null;
     var searchTextInput = this.props.enableSearch?
-      <input type='text' placeholder={this.props.searchPlaceholder?this.props.searchPlaceholder:'Search'} onKeyUp={this.handleKeyUp.bind(this)}/>:null;
+      <div className="form-group form-group-sm">
+        <input className="form-control" type='text' placeholder={this.props.searchPlaceholder?this.props.searchPlaceholder:'Search'} onKeyUp={this.handleKeyUp.bind(this)}/>
```

Comment:
Hi,

I am currently searching for a way to customize some elements like the search input.
I can see that you add the `form-control` class to it.
May be you could make it customizable in this PR ?

Something like : 
`<input className={this.props.searchClassName?this.props.searchClassName:'form-control'} type='text' placeholder={this.props.searchPlaceholder?this.props.searchPlaceholder:'Search'} onKeyUp={this.handleKeyUp.bind(this)}/>`